scripts/us_epa/air_pollution/process.py

```diff
@@ -0,0 +1,376 @@
+        self._mcf_file_path = mcf_file_path
+        self._tmcf_file_path = tmcf_file_path
+
+    def _generate_tmcf(self) -> None:
+        """
+        This method generates TMCF file w.r.t
+        dataframe headers and defined TMCF template.
+        Args:
+            None
+        Returns:
+            None
+        """
+        # Writing Genereated TMCF to local path.
+        with open(self._tmcf_file_path, 'w+', encoding='utf-8') as f_out:
+            f_out.write(_TMCF_TEMPLATE.rstrip('\n'))
+
+    def _generate_mcf(self, sv_list) -> None:
+        """
+        This method generates MCF file w.r.t
+        dataframe headers and defined MCF template
+
+        Args:
+            df_cols (list) : List of DataFrame Columns
+
+        Returns:
+            None
+        """
+
+        final_mcf_template = ""
+        for sv in sv_list:
+            sv_property = sv.split("_")
+            source = '\nemissionSource: dcs:' + sv_property[-3]
+            pollutant = '\nemittedThing: dcs:' + sv_property[-1]
+            final_mcf_template += _MCF_TEMPLATE.format(
+                pv1=sv, pv2=source, pv3=pollutant) + "\n"
+
+        # Writing Genereated MCF to local path.
+        with open(self._mcf_file_path, 'w+', encoding='utf-8') as f_out:
+            f_out.write(final_mcf_template.rstrip('\n'))
+
+    def process(self):
+        """
+        This Method calls the required methods to generate
+        cleaned CSV, MCF, and TMCF file.
+        """
+
+        final_df = pd.DataFrame(columns=['geo_Id', 'year', 'SV', 'observation'])
+        # Creating Output Directory
+        output_path = os.path.dirname(self._cleaned_csv_file_path)
+        if not os.path.exists(output_path):
+            os.mkdir(output_path)
+        sv_list = []
+
+        for file_path in self._input_files:
+            # Taking the File name out of the complete file address
+            # Used -1 to pickup the last part which is file name
+            # Read till -5 inorder to remove the .xlsx extension
+            file_name = file_path.split("/")[-1][:-5]
+            function_dict = {
+                "national_tier1_caps": _national_emissions,
+                "state_tier1_caps": _state_emissions
+            }
+            df = function_dict[file_name](file_path)
+            final_df = pd.concat([final_df, df])
+            sv_list += df["SV"].to_list()
+
+        final_df = final_df.sort_values(
+            by=['geo_Id', 'year', 'SV', 'observation'])
+        final_df['observation'].replace('', np.nan, inplace=True)
+        final_df.dropna(subset=['observation'], inplace=True)
+        final_df.to_csv(self._cleaned_csv_file_path, index=False)
+        sv_list = list(set(sv_list))
+        sv_list.sort()
+        self._generate_mcf(sv_list)
+        self._generate_tmcf()
+
+
+def main(_):
+    input_path = FLAGS.input_path
+    try:
+        ip_files = os.listdir(input_path)
+    except:
+        print("Run the download script first.\n")
```

Comment:
Can you please use log module and log.INFO here instead of print.

Reply:
changed print to logger.info